src/Index.js

```diff
@@ -652,12 +652,109 @@ Index.prototype.getSettings = function(callback) {
   var indexObj = this;
   return this.as._jsonRequest({
     method: 'GET',
-    url: '/1/indexes/' + encodeURIComponent(indexObj.indexName) + '/settings',
+    url: '/1/indexes/' + encodeURIComponent(indexObj.indexName) + '/settings?getVersion=2',
```

Comment:
Well @redox we do have to expose it some way or another, because I don't think we're the kind to ship something not backward compatible and not warn about it?

So we'll at least have to include a warning in all readmes to tell people they have to use `searchSynonyms` because now the settings don't include synonyms anymore.